about.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="stylesheets/about.css" />
+  <title>TPPS Conference 2022</title>
+</head>
+
+<body>
+  <main>
+    <header class="about-header">
+      <img class="menu" src="images/icons/menu.svg" alt="menu icon" />
+      <div class="nav-menu">
+        <ul class="menu-list">
+          <li class="mobile-nav"><a href="index.html">Home </a> </li>
+          <li class="mobile-nav"> <a href="about.html"> About</a> </li>
+          <li class="mobile-nav"><a href="#">Join</a></li>
+          <li class="mobile-nav"><a href="#">Sponsor</a></li>
+          <li class="mobile-nav"><a href="#">News</a></li>
+        </ul>
+        <div class="close">Close</div>
+      </div>
+      <div class="desktop-bar">
+
+      </div>
+      <div class="about-nav">
+        <a href="" class="logo">
+          <h1>TPPS APRIL CONFERENCE</h1>
+        </a>
+        <p class="about-description">
+         Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor, optio. Possimus reiciendis explicabo iste maxime in vitae minus, architecto aspernatur veniam ipsam voluptates natus, consectetur laboriosam ut dolores. Asperiores natus dolor mollitia animi maxime! Ut, cupiditate? Cupiditate, ut culpa dolorum excepturi cumque illum ex reiciendis harum tenetur dignissimos rem consequuntur.
+          <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem explicabo vel cumque magni molestias officia, qui architecto, voluptatem soluta cum unde? Totam, ducimus facere, quidem reprehenderit minima expedita dolorem, quis quas a rerum adipisci modi? Doloribus tenetur vitae blanditiis provident accusamus repudiandae iusto quidem, dolore, sapiente, deserunt quisquam ipsa mollitia.
+          </span>
+          <span>
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Dolorem eius obcaecati, doloribus necessitatibus consequuntur numquam, esse id totam illo voluptate repudiandae expedita facere minus? Quod debitis commodi fugiat itaque repudiandae atque facere perspiciatis, exercitationem libero nulla dolore id architecto. Mollitia sint qui nemo nobis consequuntur magnam accusantium ducimus necessitatibus eaque.
```

Comment:
[OPTIONAL]
- Some of your text is overlapping which does not look too good. If you have the time, could you fix that?
![image](https://user-images.githubusercontent.com/45460192/196790116-8d99d378-9ed1-4a2e-a6f6-c88a4f2432ff.png)